release-1.6/release-1.6.md

```diff
@@ -0,0 +1,42 @@
+(To Be Approved at upcoming community meeting)
+
+#Proposed timeline for 1.6
+Since Kubernetes 1.5 is set to release on Dec 12, 2016, and 
+December tends to have a lot of vacation time, we will begin the 1.6 release the first week of 2017.
+We will use the time between releases for planning, designing and bugfixes.
+
+
+The proposal below is identical in layout to the 1.4 and 1.5 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- some sigs will focus on stabilization for 1.6, and there may be fewer new
+  feature proposals
+- there are fewer major holidays and vacation breaks this quarter 
+
+##1.6 Overview
+- Dec 8 - Jan 3: planning/design/bugfix period
+- Jan 3 (Tues) coding start (7w)
+- Jan 24 (Tues) features repo freeze
```

Comment:
yeah, no new features are tagged v1.6 and that's when proposed features should be in the features repository with the tag.  Exception features is for features that need to get in past the start of code feature freeze.